flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -29,13 +30,27 @@ empty_weight_yaw_MOI = 3507948 ; Empty yaw moment of inertia, Jyy (SLUG SQ FEET)
 empty_weight_coupled_MOI = 1000 ; Empty transverse moment of inertia, Jyz (SLUG SQ FEET)
 activate_mach_limit_based_on_cg = 0 ; Activate mach limitation depending on CG position (true if > 0 /false othewise). Mostly for Concorde).
 activate_cg_limit_based_on_mach = 0 ; Activate cg limitation depending on mach value (true if > 0 /false othewise). Mostly for Concorde).
-max_number_of_stations = 6 ; Number of payload stations
-station_load.0 = 170, 35.59, -1.86, 5.017127, TT:MENU.PAYLOAD.PILOT, 1
-station_load.1 = 170, 35.59, 1.86, 5.017127, TT:MENU.PAYLOAD.COPILOT, 2
-station_load.2 = 6115, 20.845893, 0, 5.039792, TT:MENU.PAYLOAD.BUSINESS_CLASS, 0
-station_load.3 = 3505, 21.825772, 0, 0.94849, TT:MENU.PAYLOAD.FORWARD_BAGGAGE, 0
-station_load.4 = 4000, -21.223969, 0, 5.794459, TT:MENU.PAYLOAD.ECONOMY_CLASS, 0
-station_load.5 = 6310, -35.825745, 0.000001, 1.007935, TT:MENU.PAYLOAD.REAR_BAGGAGE, 0
+; weight & balance for A320-251N (WV055) - 174 economy class seats (Azul A20N)
+; MRamp = 79400 kg/175047 lbs
+; MTOW = 79000 kg/174165 lbs - should match line 1 above
+; MLDW = 67400 kg/148591 lbs
+; MZFW = 64300 kg/141757 lbs
+; OEW = 46262 kg/101990 lbs
+; Max Payload = 23295 kg/51357 lbs
+; Max Fuel = 23723 ltrs (19045 kg)/6267 us gal (41989 lbs) @ specific gravity of 0.8 ISA conditions
+; Average Adult pax (male + female combined) assumed to be 185lbs/84kg
```

Comment:
Suggest adding a comment here:  ";Passenger baggage weight of 20 kg per pax is also added, making total pax + baggage weight = 104 kg/pax (=104 kg/pax passenger weight on Simbrief)" to avoid confusion over what is appropriate for the Simbrief profile.

Reply:
Added 👍🏻